src/pages/[platform]/build-a-backend/graphqlapi/connect-api-to-existing-database/index.mdx

```diff
@@ -0,0 +1,670 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
```

Comment:
there should also be a directory.mjs file change that puts this into the doc

Reply:
Added page to directory.mjs